test/fuzz/lua/test_engine.lua

```diff
@@ -485,10 +582,87 @@ local function format_op(space, space_format)
     space:format(space_format)
 end
 
+local function scan_column_op(space, fields, key)
+    if space.engine ~= 'memcs' then
+        return
+    end
+    local mpkey = msgpack.encode(key)
+    local scanner = column_scanner.box_index_scanner(space.id, 0, fields, mpkey)
+    while true do
+        if math.random(1, 20) == 1 then
+            log.info('SCAN_COLUMN: finish early')
+            break
+        end
+        local max_batch_size = oneof({10, 10^6})
```

Comment:
I choose here between a small batch (up to 10 elements) and a large batch (up to 10^6 elements) so that small batches appear much more frequently. If I used `math.random(10^6)`, there would be only `1/10^5` probability of a small batch.

Reply:
please add a comment